### PR TITLE
[Time Conductor] Prevent route change when time conductor values change

### PR DESCRIPTION
**Pull request by akhenry**
Addresses #1341 

### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

### CODE REVIEW

**Comment by larkin** [comment]
I saw this solution Friday and was a bit boggled with the solution.  Routes were configured as expected (`reloadOnSearch: false`) but they still weren't navigating properly.

f20701e...0b543ba resolves the underlying issue and fixes the issue more permanently, and also removes the requirement to workaround the odd angular behavior.  It causes angular to trigger route updates instead of trying to trigger route changes and failing.

It also somewhat simplifies the browsecontroller logic.  I also cleaned up the specs and am using real async tests instead of fakePromise, which was causing invalid behaviors.

One change here that is slightly out of scope for the original issue is I changed the InstantiateCapability such that it contextualizes the object that is instantiated so that the object has a path and can be navigated to immediately afterwards for editing.  @akhenry @VWoeltjen if you know of any complex behaviors this might have problems due to this, let me know. As far as I can tell, it works just fine.

# Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

File: platform/features/conductor-v2/conductor/src/ui/TimeConductorController.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ define(
          * @memberof platform.features.conductor
          * @constructor
          */
-        function TimeConductorController($scope, $window, $location, openmct, conductorViewService, timeSystems, formatService) {
+        function TimeConductorController($scope, $window, $location, $route, openmct, conductorViewService, timeSystems, formatService) {
```
Comment (VWoeltjen) [inline platform/features/conductor-v2/conductor/src/ui/TimeConductorController.js]: `$route` looks like an unused dependency here

File: platform/commonUI/browse/bundle.js [hunk 2]
```diff
@@ -72,14 +72,13 @@ define([
...
-                    "when": "/browse/:ids*",
+                    "when": "/browse/:ids*?",
                     "template": browseTemplate,
                     "reloadOnSearch": false
                 },
                 {
                     "when": "",
-                    "template": browseTemplate,
-                    "reloadOnSearch": false
+                    "redirectTo": "/browse/"
```
Comment (VWoeltjen) [inline platform/commonUI/browse/bundle.js]: OH I SEE!

File: platform/features/conductor-v2/conductor/bundle.js [hunk 3]
```diff
@@ -68,6 +68,7 @@ define([
                         "$scope",
                         "$window",
                         "$location",
+                        "$route",
```
Comment (VWoeltjen) [inline platform/features/conductor-v2/conductor/bundle.js]: Change could be removed since `$route` is unused

**Comment by VWoeltjen** [comment]
> One change here that is slightly out of scope for the original issue is I changed the InstantiateCapability such that it contextualizes the object that is instantiated so that the object has a path and can be navigated to immediately afterwards for editing. @akhenry @VWoeltjen if you know of any complex behaviors this might have problems due to this, let me know. As far as I can tell, it works just fine.

Didn't address this in review comments, but this behavioral change makes sense to me. I don't expect it should cause problems elsewhere (I can't imagine any of our behavior being contingent on _not_ having context)

**Comment by akhenry** [comment]
> I changed the InstantiateCapability such that it contextualizes the object that is instantiated 

Great! There's been an issue for this lurking around for ages actually - https://github.com/nasa/openmct/issues/575

**Comment by larkin** [comment]
@VWoeltjen @akhenry I've addressed changes from review.  This needs a complete reviewer checklist prior to a merge.

**Comment by akhenry** [comment]
### Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? Y
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards? Y

Although I am reviewing and merging a PR that I opened, the changes were actually made by @larkin.

**Comment by larkin** [comment]
Rebased to remove conflicts.  @akhenry go wild.

**Pull request merged**